odilia/src/main.rs

```diff
@@ -90,13 +90,21 @@ async fn sigterm_signal_watcher(
 
 use atspi::events::document::LoadCompleteEvent;
 
+#[tracing::instrument]
+async fn change_priority(
```

Comment:
this function shouldn't be here imo, we don't change the priority at runtime, we should have implied or fix priorities for message categories, changeable from config files, this however just bloats main.rs imo

Reply:
As discussed, we should set the priority for each speakable piece of text. You are right that it should not be its own function though.